### PR TITLE
KAFKA-13231; `TransactionalMessageCopier.start_node` should wait until the process if fully started

### PR DESCRIPTION
**Pull request by dajac**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by dajac** [comment]
Transactions system tests: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/4663/

**Comment by dajac** [comment]
> Transactions system tests: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/4663/

All tests have passed: http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2021-08-26--001.system-test-kafka-branch-builder--1629974240--dajac--KAFKA-13231--584309932/report.html

**Review by hachikuji — APPROVED** [review]
LGTM

File: tests/kafkatest/services/transactional_message_copier.py [hunk 1]
```diff
@@ -159,12 +159,16 @@ def pids(self, node):
     def alive(self, node):
         return len(self.pids(node)) > 0
 
+    def start_node(self, node):
+        BackgroundThreadService.start_node(self, node)
+        wait_until(lambda: self.alive(node), timeout_sec=60, err_msg="Message Copier failed to start")
```
Comment (hachikuji) [inline tests/kafkatest/services/transactional_message_copier.py]: Is it worth adding `node` to these error messages?

**Comment by dajac** [comment]
@hachikuji updated the PR to add the node to the error message.

**Comment by dajac** [comment]
New run to validate: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/4665/

**Review by hachikuji — APPROVED** [review]

**Comment by dajac** [comment]
Successful run: http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2021-08-26--001.system-test-kafka-branch-builder--1630007187--dajac--KAFKA-13231--7007d8ec3/report.html

**Comment by dajac** [comment]
Merged to trunk, 3.0 and 2.8.

**Pull request merged**